soccer/planning/planner/LineKickPlanner.hpp

```diff
@@ -1,46 +1,43 @@
 #pragma once
 
-#include "Planner.hpp"
-#include "planning/Instant.hpp"
-#include "planning/low_level/Replanner.hpp"
+#include <optional>
+
+#include "planning/Trajectory.hpp"
+#include "planning/planner/Planner.hpp"
+
+class Configuration;
+class ConfigDouble;
 
 namespace Planning {
 
 /**
- * @brief Planner to kick a (possibly moving) ball without stopping to pivot
- * around it. The robot will move "through" the ball, and kick just as it comes
- * into contact with it.
+ * Planner which plans a path to line kick a ball.
+ * Uses the System State object to get the position of the ball
+ * and predict its Path. It chooses the closest intersection point
+ * with the ball Path it can reach in time and plans a Path so the
+ * ball and robot intersect at the same time.
+ *
+ * TODO(Kyle): Overhaul this entire planner. It's sketchy right now.
```

Comment:
Whoops forgot this todo

Reply:
Honestly we don't even really have a good idea of what we want to do with it, so I'm not comfortable making an issue for it yet.

Reply:
I think it'd be nice to make an issue to put it somewhere on the todo list so that we don't forget that we have a sketchy `LineKickPlanner`